hoomd/mpcd/DryingDropletStreaming.h

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2009-2019 The Regents of the University of Michigan
+// This file is part of the HOOMD-blue project, released under the BSD 3-Clause License.
+
+// Maintainer: mphoward
+
+/*!
+ * \file mpcd/DryingDropletStreaming.h
+ * \brief Declaration of mpcd::DryingDropletStreaming
+ */
+
+#ifndef MPCD_DRYING_DROPLET_STREAMING_METHOD_H_
+#define MPCD_DRYING_DROPLET_STREAMING_METHOD_H_
+
+#ifdef NVCC
+#error This header cannot be compiled by nvcc
+#endif
+
+#include "ConfinedStreamingMethod.h"
+#include "hoomd/extern/pybind/include/pybind11/pybind11.h"
+#include "hoomd/Variant.h"
+#include "BoundaryCondition.h"
+
+namespace mpcd
+{
+
+//! MPCD confined streaming method
```

Comment:
Don't duplicate documentation from the base class, retain only the things that are special to this class.

This class also does not have a `\tparam` (template parameter) since it is specialized to the `SphereGeometry`.